source/codegen/metadata/nifake/functions.py

```diff
@@ -1176,6 +1176,41 @@
         'returns': 'ViStatus',
         'use_session_lock': False
     },
+    'GetPatternPinIndexes':{
```

Comment:
There's also `GetAnIviDanceWithATwistString` in NIFake. Consider either deleting it to rely on this new one or updating it so it generates as expected and add a test for it.

Reply:
I removed this.